includes/parser/Parser_LinkHooks.php

```diff
@@ -55,7 +55,6 @@ function firstCallInit() {
 
 		wfProfileIn( __METHOD__ );
 
-		$this->setHook( 'pre', array( $this, 'renderPreTag' ) );
```

Comment:
The `renderPreTag` no longer exists here. And is registered here:
https://github.com/Wikia/app/blob/db60078cc6bb4185814bdd4df8a957be0889cf68/includes/parser/CoreTagHooks.php#L19-L19